lib/extend/device-info.js

```diff
@@ -0,0 +1,37 @@
+import { Dimensions, PixelRatio } from 'react-native';
+
+class DeviceInfo {
+    constructor() {
+        const { width, height } = Dimensions.get('window');
+
+        this.pixelDensity = PixelRatio.get();
+        this.width = width;
+        this.height = height;
+        this.adjustedWidth = this.width * this.pixelDensity;
+        this.adjustedHeight = this.height * this.pixelDensity;
+        this.isPortrait = height >= width;
```

Comment:
If I understand correctly, if an app is responsive (can be flipped from portrait to landscape), we will determine this setting only on launch and won't be responsive to changes.
Don't you think it can be confusing to the users?

If someone sets ignore on `LANDSCAPE`, launches on landscape and then flips to portrait, size-matters will still be disabled, I think it's a big problem with this feature..